cityofaustin/atd-moped
# displays project lead in list view, searchable

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10683

## Testing
**URL to test:** 

https://10683_search_based_on_lead_workgroup--atd-moped-main.netlify.app/moped/

**Steps to test:**
- go to a project and add a 'lead'
- go to project list view, you should see the 'lead' to the left of 'sponsor'
- search by that 'lead' and you should see your record come up

---
#### Ship list
